app/code/Magento/CatalogInventory/Model/Product/QuantityValidator.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright 2024 Adobe
+ * All Rights Reserved.
+ */
+declare(strict_types=1);
+
+namespace Magento\CatalogInventory\Model\Product;
+
+use Magento\CatalogInventory\Api\StockRegistryInterface;
```

Comment:
interface StockRegistryInterface has been deprecated. Please use alternate.

Reply:
Yes @Mohamed-Asar you can use that. BTW its now recommended to disable the MSI module.

Reply:
@engcom-Hotel MSI module is not included in this repo. how do i proceed ?

Reply:
Please refer to this repo: https://github.com/magento/inventory

Reply:
@engcom-Hotel I used MSI and remvoed the deprecated methods, please review